packages/vuetify/src/components/VWindow/VWindow.ts

```diff
@@ -222,7 +222,10 @@ export default BaseItemGroup.extend({
       this.internalValue = this.getValue(item, lastIndex)
     },
     updateReverse (val: number, oldVal: number) {
-      const lastIndex = this.items.length - 1
+      const itemsLength = this.items.length
+      const lastIndex = itemsLength - 1
+
+      if (itemsLength <= 2) return val < oldVal
```

Comment:
@nekosaur , please help to check if this change is compatible with fixing issue #11280 from this PR.

Thank you in advance :)